.github/workflows/benchmark.yml

```diff
@@ -2,6 +2,19 @@ name: Benchmark this PR
 on:
   pull_request_target:
     branches: [ main ]  # change to your default branch
+    paths-ignore:
+      - 'CITATION.bib'
+      - 'LICENSE.md'
+      - 'README.md'
+      - '.zenodo.json'
+      - '.github/workflows/benchmark.yml'
```

Comment:
```suggestion
```
I think usually we want to run benchmarks if we change the benchmark action file itself.